validation.js

```diff
@@ -0,0 +1,34 @@
+const errorSpan = document.getElementById('error');
+const email = document.getElementById('email');
+const form = document.getElementById('contact-form');
+
+function validateEmail() {
+  const lowerCase = /^[a-z0-9.@]+$/;
+  const result = lowerCase.test(email.value);
+  return result;
+}
+
+function showError() {
+  errorSpan.style.display = 'block';
+  errorSpan.textContent = 'Email address must be lowercase';
+}
```

Comment:
- [ ] Well done on implementing the function to show error but kindly fix the issue because when I click submit with an email with uppercase it is not showing anything. Keep coding 💪

Reply:
hello @Donard97 
am sorry but it works locally. and when am try to push just to make sure that every thing local is on github also I get the message every thing up to date. 😄

Reply:
hey @Donard97 
how can I see the code live, so I can check the form at github.
need your help please 😃